eip-0006.md

```diff
@@ -1,4 +1,4 @@
-# EIP-0006: Informal Multi-Stage Contract Specification Format
+# EIP-0006: Informal Multi-Stage Specification Format
```

Comment:
In addition let's change "Specification" to "Description" in the title to reflect what you described as "it were an essay rather than the clunky feeling of writing compiled code (or a formal spec)". 
And also change Format to Template.
So I think of title "EIP-0006: Informal Multi-Stage Contract Description Template".

Reply:
Also, looking at the list of EIPs it makes me feel confused, because both the  Contract Template and the  Well-known Contracts EIPs use the "contract" term to mean "script".
While this EIP use Contract to mean the whole scenario.

Reply:
I think "Multi-Stage Protocols" works pretty well which is what scalahub sugested on Discord.

Reply:
I disagree with changing it to "Description", because these are informal specifications of multi-stage protocols. This isn't just a loose "description" which states how a protocol would work, but instead is a specification which follows a given semi-formal format that is written in plain english, thus being informal. The vast majority of specifications in software are informal/semi-formal, and none of them are labled as "descriptions", and I believe that changing to it only makes it more vague and less understandable.